pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "svgdigitizer"
+license = {text = "GPL 3.0+"}
+description = "svgdigitizer is a Python library and command line tool to recover the measured data underlying plots in scientific publications."
+requires-python = ">=3.9"
+dependencies = [
+    "astropy>=5,<6",
+    "click>=8,<9",
+    "frictionless>=5.8.3,<6",
+    "matplotlib>=3.5,<4",
+    "mergedeep>=1.3.4,<2",
+    "pandas>=1.3,<2",
+    "pdf2image>=1.16,<2",
+    "pillow>=10.0.1,<11",
+    "pybtex>=0.24,<0.25",
+    "pyyaml>=6,<7",
+    "scipy>=1.7,<2",
+    "svg.path>=4.1,<5",
+    "svgpathtools>=1.4,<2",
+    "svgwrite>=1.4,<2",
+]
+dynamic = ["version"]
+
+[tool.setuptools.dynamic]
+version = {attr = "svgdigitizer.__init__.__version__"}
```

Comment:
I'd just hardcode the version number here and update it with the `$VERSION_BUMP_PATTERNS` in rever.